resources/dput.cf

```diff
@@ -1,4 +1,4 @@
-[all-bootstrap]
+[ros_bootstrap]
```

Comment:
I think underscores are invalid, otherwise I would have used them already. I think they need to be dashes. For consistency I'd rename it the same below. 

In the documentation this element is the "host" and I think they use hostname validation.